lib/kubeclient/common.rb

```diff
@@ -136,20 +136,7 @@ def watch_entities(entity_type, resource_version = nil)
         uri.query = URI.encode_www_form('resourceVersion' => resource_version)
       end
 
-      options = {
```

Comment:
Yes.  The new `#watch_pod_log` method needs these options as well, so I extracted them out into the `#net_http_options` method to DRY up the code.